server/routes/createPlan/checkYourAnswers/checkYourAnswersController.test.ts

```diff
@@ -80,31 +102,44 @@ describe('CheckYourAnswersController', () => {
     })
 
     it('On error - Calls next with error', async () => {
+      // Given
       mockService.createCiagPlan.mockImplementation(() => {
         throw new Error('mock_error')
       })
 
-      controller.post(req, res, next)
+      // When
+      await controller.post(req, res, next)
 
+      // Then
       expect(next).toHaveBeenCalledTimes(1)
       expect(res.render).toHaveBeenCalledTimes(0)
     })
 
     it('On success - UPDATE - Calls api update', async () => {
+      // Given
       req.context.plan = { hopingToGetWork: HopingToGetWorkValue.NOT_SURE, desireToWork: false }
       setSessionData(req, ['isUpdateFlow', id], true)
       mockService.updateCiagPlan.mockReturnValue({})
-      controller.post(req, res, next)
 
-      expect(mockService.updateCiagPlan).toBeCalledTimes(1)
+      // When
+      await controller.post(req, res, next)
+
+      // Then
+      expect(mockService.updateCiagPlan).toHaveBeenCalledTimes(1)
```

Comment:
I'd like to see this assertion assert the data that was actually sent to the `updateCiagPlan` method, but at least now we are also asserting the URL that we redirect to 🤷